### PR TITLE
Add remaster changes to long jump action macro

### PR DESCRIPTION
**Pull request by nikolaj-a**
Update the text with the default difficulty class to reflect the text in Player Core, and assign the default difficulty class for the action macro when no other DC is provided.

### CODE REVIEW

File: src/module/system/action-macros/athletics/long-jump.ts [hunk 1]
```diff
@@ -14,7 +14,7 @@ export function longJump(options: SkillActionOptions): void {
         traits: ["move"],
         event: options.event,
         callback: options.callback,
-        difficultyClass: options.difficultyClass,
+        difficultyClass: options.difficultyClass ?? { value: 15 },
```
Comment (stwlam) [inline src/module/system/action-macros/athletics/long-jump.ts:17]: Why a default?

Comment (nikolaj-a) [reply src/module/system/action-macros/athletics/long-jump.ts:17]: That’s how it works now. It’s like High Jump, except with DC 15 instead of DC 30.

Comment (nikolaj-a) [reply src/module/system/action-macros/athletics/long-jump.ts:17]: > You Stride, then attempt a DC 15 Athletics check to make a long jump in the direction you were Striding. If you didn’t Stride at least 10 feet, you automatically fail your check. The GM might increase or decrease this DC depending on the situation.

Comment (stwlam) [reply src/module/system/action-macros/athletics/long-jump.ts:17]: Lame!

**Pull request merged**